### ISSUE TITLE
Allow color temperature control through lightener light

### ISSUE THREAD

**Issue by joshmwilliams** [fredck/lightener#6]
Could this be updated to allow control of child-lights color-temperature through the parent light? 

Currently you cannot use both adaptive-lighting (Or circadian lighting) as it'll fight to update the brightness of the children lights. You can disable the children in adaptive-lighting but then you lose the color temperature updates. If this plugin would allow passthrough color temperature, you could remove children lights from adaptive lighting and just allow control of the parent light

**Comment by fredck** [comment]
I believe this goes beyond color temperature.

Right now we control the "brightness" of the lights, which is in fact the core feature of Lightener. But it is true that Lightener could be seen as a replacement for the "Group" feature and therefore concentrate and passthrough all the other features available for light entities.

Apart from brightness and on-state, all the other [features of lights](https://developers.home-assistant.io/docs/core/entity/light#properties) are related to color. I believe the proper implementation should follow this:

  * Enable on the controlling Lightener all features available on *any* of the controlled lights.
  * When an enabled feature is changed on the Lightener, it pass the change along to all controlled lights that support such feature.

On this is implemented, in a follow up feature request, we could consider making it possible to configure which controlled lights should inherit color changes, to offer a better level of control.

Does all that make sense?

**Comment by joshmwilliams** [comment]
Makes sense and I agree with the approach.

**Comment by ironcrafter54** [comment]
I agree this would be very helpful for use with adaptive lighting. The solution @joshmwilliams would be to set up 2 instances of adaptive lighting, one for the color tempature control, where you select each individual light, or a traditional light group, and the other would be just for brightness. Then you would have to link the sleep mode and adaptive light options using a switch group.

**Comment by fredck** [comment]
I'v just released [v2.1.0](https://github.com/fredck/lightener/releases/tag/v2.1.0). I'm very curious what you guys have to say #give-feedback :)

**Comment by jimmhead** [comment]
@joshmwilliams did this end up working for you?
I'm in the same scenario, but I cannot find any way for adaptive lighting and Lightener to work together.

@fredck adaptive lighting is somehow passing the brightness controls to the child lights through the lightener entity. So with nothing but the Lightner entity added to adaptive lighting, when adaptive lighting adjusts the lightener entity, the lightener adjustments immediately get overridden by the adaptive lighting adjustments, any ideas?
